app/build.gradle

```diff
@@ -14,6 +14,11 @@ android {
         buildConfigField "String", "mail_host", "\"smtp.gmail.com\""
         buildConfigField "String", "weather_api_key", "\"852c6dcc0bd878109bd0fc7bb1566c3e\""
         buildConfigField "String", "weather_icon_url", "\"https://openweathermap.org/img/w/\""
+        buildConfigField "String", "marvel_api_key_public", "\"08cd7f2cd10462f4d20ab91d76ecec33\""
```

Comment:
All the constants here should be in UPPER_CASE.

Reply:
Done